src/drc/datamodel/tests/factories.py

```diff
@@ -18,6 +18,10 @@ class EnkelvoudigInformatieObjectFactory(factory.django.DjangoModelFactory):
     formaat = 'some formaat'
     taal = 'dut'
     inhoud = factory.django.FileField(data=b'some data', filename='file.bin')
+    link = 'http://www.example.com/'
```

Comment:
is het niet beter om in factories enkel velden te genereren die niet leeg mogen zijn? URL kan je zelf altijd nog specifieren als je die nodig hebt, maar lege url is valide data